src/components/SearchBox/SearchBoxComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Grid, Container, TextField, Button } from "@material-ui/core";
+import { searchBoxStyle } from "../../styles/styleSheet";
+
+const SearchBox = ({
+  dispatchGetSearch,
+  dispatchStopSearch,
+  dispatchGetMovieListAction,
+}) => {
+  const styles = searchBoxStyle();
+  const handleChange = (v) => {
```

Comment:
Please use meaningful names.

Reply:
i have the character "v" with meaning full name